src/Widget/WidgetInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Widget;
+
+use Symfony\Component\HttpFoundation\Response;
+
+interface WidgetInterface
+{
+    public function getName(): string;
+
+    public function getType(): string;
+
+    public function getTarget(): string;
```

Comment:
add some comments here, like this should return const from Target enum

Reply:
Done, and did the same for `getZone`.